airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/avro/JsonToAvroSchemaConverter.java

```diff
@@ -201,7 +201,6 @@ Schema getSingleFieldType(final String fieldName,
         if (fieldDefinition.has("format")) {
           final String format = fieldDefinition.get("format").asText();
           fieldSchema = switch (format) {
-            case "timestamp-micros" -> LogicalTypes.timestampMicros().addToSchema(Schema.create(Schema.Type.LONG));
```

Comment:
This is a hack that should not be added in the first place. Json schema does not have such format. This is an Avro format.